test/util/config/directory.go

```diff
@@ -0,0 +1,34 @@
+package config
```

Comment:
Move these files in `pkg/fakerp` then you won't need to be importing them in a bunch of places.

Reply:
done